src/main/java/de/refactoringbot/refactoring/supportedrefactorings/RemoveMethodParameter.java

```diff
@@ -134,7 +134,8 @@ public String performRefactoring(BotIssue issue, GitConfiguration gitConfig) thr
 
 		// If refactor-method not found
 		if (methodToRefactor == null) {
-			throw new BotRefactoringException("Could not find specified method! Automated refactoring failed.");
+			throw new BotRefactoringException(
+					"Could not find method declaration of specified parameter at given line! Please check if you placed your comment on the method declaration of the parameter that needs to be removed.");
```

Comment:
Same issue with the second part of the message as already mentioned in a previous comment.